### PR TITLE
Fixes #36968 - Add container manifest list to UI and aria-label for repo icon

### PR DESCRIPTION
**Pull request by sjha4**
#### What are the testing steps for this pull request?
Create a docker repo with manifest lists. ex: libpod/testimage from quay.io
Sync repo and have the repo sync to a capsule
Check the capsule content page to make sure you see manifest lists in content counts.
Also do an inspect element on repo type icon. It should have aria-lebel = Repo type

### CODE REVIEW

File: webpack/scenes/SmartProxy/AdditionalCapsuleContent.js [hunk 1]
```diff
@@ -55,6 +56,11 @@ const AdditionalCapsuleContent = ({ counts }) => {
         {`${dockerManifestCount} Container manifests`}<br />
       </>
             }
+      {dockerManifestListCount > 0 &&
+      <>
+        {`${dockerManifestListCount} Container manifest lists`}<br />
```
Comment (ianballou) [inline webpack/scenes/SmartProxy/AdditionalCapsuleContent.js:61]: One thing I realized -- it doesn't look like the container bits are getting translated (except ansible collections?) for some reason:

image

vs

image

**Review by ianballou — COMMENTED** [review]
I just realized that content view versions don't list manifest lists either!

image

**Comment by sjha4** [comment]
Added https://projects.theforeman.org/issues/36988 for adding the CV version container manifest list counts and comparisons..

**Review by ianballou — APPROVED** [review]
Translations work now!

**Pull request merged**